model-optimizer/mo/front/onnx/loader.py

```diff
@@ -109,7 +109,7 @@ def protobuf2nx(graph: Graph, pb):
                     'out': src_port,
                     'in': 0,
                     'name': out,
-                    'fw_tensor_debug_info': [(id, src_port, out)],
+                    'fw_tensor_debug_info': [(node_id(node), src_port, out)],
```

Comment:
Why this change is needed? The `id` is already the uniquely generated id of the graph node (line 77). And this `id` is actually the right identifier of the node which should be stored in the fw_tensor_debug_info

Reply:
As we discussed we need to use node_id(node). Refactored this code for better readability.